common/models/preview-task.js

```diff
@@ -1,3 +1,31 @@
 module.exports = function(PreviewTask) {
-
+    PreviewTask.claim = function(renderer, cb) {
+    PreviewTask.findOne({where: {renderer: renderer, startTime: null}},
+     function (err, model) {
+          if (err) {
+            console.error(err);
+          }
+          else {
+            if (model != null) {
+              model.updateAttribute('startTime', new Date().getTime() / 1000);
+              // seconds since midnight, 1 Jan 1970
+              model.updateAttribute('expireTime',new Date().getTime()/1000 + 2*1000);
```

Comment:
So am I reading correctly that the `2*1000` means the task will expire after ~33 minutes? (This numerical value which defines how long until expiration is often called "time-to-live" or "TTL".)

Handling a numerical constant like `2*1000` works but is usually poor form. (I'm guilty of doing things like this, too, especially when learning a new framework.) There's some good discussion at https://en.wikipedia.org/wiki/Magic_number_%28programming%29#Unnamed_numerical_constants

The simplest fix is to move to a named variable/constant (eg `var TASK_TTL = 2*1000;` or better yet `var TASK_TTL = 30 * 60;`).